client/app/locales/en.json

```diff
@@ -6,7 +6,7 @@
   "notification homes": "%{smart_count} home description imported |||| %{smart_count} home descriptions imported",
   "notification consumptionstatements": "%{smart_count} new consumption statement imported |||| %{smart_count} new consumption statements imported",
   "notification contracts": "%{smart_count} new contract imported |||| %{smart_count} new contracts imported",
-  "notification clients": "%{smart_count} new client's informations imported",
+  "notification clients": "%{smart_count} new client's information imported |||| %{smart_count} new clients' information imported",
```

Comment:
Nope, it means "clients's information" and you don't do that in english. In that case the possessive "s" disappears.